docs/source/installation.rst

```diff
@@ -59,10 +59,10 @@ Option 4: Installing via Docker
 -------------------------------
 
 Chemprop can also be installed with Docker, making it possible to isolate the Chemprop code and environment.
-To install and run Chemprop in a Docker container, first install `Docker`_.
+To install and run Chemprop in a Docker container, first `install Docker`_.
```

Comment:
Why was this required?

Reply:
Previously in this file (line 6), the `Docker` reference is defined and points to the Docker "Get Started" page, whereas here we want it to point to the Docker "Get Docker" page. I think the easiest way to accommodate was just to make these two separate references.

Reply:
Thanks for catching that! I'm surprised that wasn't an error.

Reply:
It does give an error, but one that's recoverable, see the [current install page ](https://chemprop.readthedocs.io/en/latest/installation.html) and how the first \`Docker\` ref is messed up.
I caught these errors because I parsed through the error log during compilation - usually this isn't seen because the docs are compiled automatically, but I was compiling on my local device.